### DISCUSSION TITLE
Application Error The page was not found in the hosted application

### DISCUSSION THREAD

**Discussion by lete114** [vercel/vercel#5199] [Help]
Just yesterday, there was a problem with my website as shown in the figure below. I did not do any operation on the website four days before the problem occurred, but the website was normal after I clicked redeploy. I wonder what is the reason for this
![image.png](https://upimage.alexhchu.com/2020/09/16/0f61cf8485dee.png)

**Comment by paulogdm** [comment]
It is possible that deployment was taken down due to a [Fair Use Policy](https://vercel.com/docs/platform/fair-use-policy) breach. Please check your SPAM folder and reach out to our support for further information.
